test/integration/serial_test.rb

```diff
@@ -56,5 +68,23 @@ class SerialTest < ActionDispatch::IntegrationTest
       title = find ".serial-content__title"
       assert_equal(@serials.last.title, title.text)
     end
+
+    test "serial description" do
+      serial = create(:serial, :with_posts, title: "serial title", site: @site, description: <<~DESC)
+      # h1
+
+      contents
+      DESC
+      visit "/serials"
+
+      click_on serial.title
+
+      within ".serial-content__description" do
+        h1 = find "h1"
```

Comment:
`find` is Capybara DSL, I think.
So I don't use `find` method without parentheses.

According to our coding standard. See esa, please.